docs/guide/transformers.md

```diff
@@ -1,7 +1,295 @@
-# Document Transformers
+# Transformers
 
 !!! note "Transformers are optional, not mandatory"
-
     Graph traversal operates on the structured metadata.
-    Document transformers provide tools for populating the metadata, but they are not necessary.
-    In many cases you may have existing structred information that is useful in addition or instead of what the transformers would populate.
\ No newline at end of file
+    Transformers provide tools for populating the metadata, but they are not necessary.
+    In many cases you may have existing structured information that is useful
+    in addition or instead of what the transformers would populate.
+
+We provide two types of document transformers that can be useful in setting up your
+documents for graph traversal.
+
+* **Information Extractors:** These extract information out of document content
+    and add to the metadata.
+
+* **Metadata Utilities:** These add to or modify document metadata to enable certain
+    features
+
+## Information Extractors
+
+!!! note "Extras required"
```

Comment:
I don't know that I love how this pops up in the flow of reading. It feels very abrupt. Could it be done as a tooltip or footnote? Or possible using `???` to make it collapsible?

https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks